apps/openmw/mwworld/weather.cpp

```diff
@@ -427,29 +429,33 @@ void WeatherManager::update(float duration, bool paused)
     else
         mRendering->getSkyManager()->sunEnable();
 
-    // sun angle
-    float height;
-
-    //Day duration
-    float dayDuration = (mNightStart - 1) - mSunriseTime;
+    // Update the sun direction.  Run it east to west at 15 degrees south from overhead.
+    // The sun's speed at day and night will differ, since mSunriseTime and mNightStart
+    // mark when the sun is level with the horizon.
+    {
+        assert( mNightStart > mSunriseTime );
```

Comment:
You might want to consider changing this to an if/throw block rather than an assert.